src/pages/contact-list/contact-list.ts

```diff
@@ -17,21 +18,29 @@ export class ContactListPage {
   searchQuery: string = '';
   //For tests
   contacts:any;
+  originalContact: any;
+  token: string;
   verif0Contact: boolean = false;
 
-  constructor(public navCtrl: NavController, public navParams: NavParams, public menuCtrl: MenuController, public contactServices: ContactServicesProvider) {
+  constructor(public navCtrl: NavController, public navParams: NavParams, public menuCtrl: MenuController, public contactServices: ContactServicesProvider, public userServices: UserServicesProvider) {
     this.menuCtrl.enable(true);
 
+    this.token = this.userServices.token
```

Comment:
this is not useful since you can access userServices token easily on any page, by adding the provider on constructor

Reply:
fixed